kftlib/examples/get_functions_times.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+"""
+Example application using kftlib.
+"""
+
+import argparse
+import statistics
+import os
+
+from kftlib.elf import Elf
+from kftlib import inspect
+from kftlib.stats import get_functions_times
+
+from pathlib import Path
+
+
+def get_fn_times(events, elf, functions, out):
+    fn_pcs = list(map(lambda fn: elf.fun2pc.get(fn), functions))
+    fn_times = get_functions_times(events, fn_pcs)
+
+    sumt = 0
+    print(f"{'function':>13}: {'count':>5} {'avg time':>8}")
+    for fn, pc in zip(functions, fn_pcs):
+        if not pc:
+            continue
+        if pc in fn_times:
```

Comment:
If the code block below does the bulk of body function, then please reverse this condition and add it to the code that continues the loop.